<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Motion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Stored Positions</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="home_pushButton">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="park_pushButton">
        <property name="text">
         <string>Park</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="set_stored_position_comboBox">
        <item>
         <property name="text">
          <string>Test</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TEst2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Ground</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="5">
       <widget class="QDoubleSpinBox" name="yaw_target_position_doubleSpinBox"/>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="pitch_position_doubleSpinBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="y_target_position_doubleSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="y_position_doubleSpinBox"/>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="z_target_position_doubleSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Actual</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="x_position_doubleSpinBox"/>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="z_position_doubleSpinBox"/>
      </item>
      <item row="2" column="4">
       <widget class="QDoubleSpinBox" name="pitch_target_position_doubleSpinBox"/>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="yaw_position_doubleSpinBox"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="x_target_position_doubleSpinBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X
optic axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QPushButton" name="read_current_position_pushButton">
        <property name="text">
         <string>Read Current Position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="live_checkBox">
        <property name="text">
         <string>Live</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
